scene/main/viewport.cpp

```diff
@@ -2944,7 +2944,18 @@ void Viewport::unhandled_input(const Ref<InputEvent> &p_event, bool p_local_coor
 	get_tree()->_call_input_pause(unhandled_input_group, "_unhandled_input", ev, this);
 	//call_group(GROUP_CALL_REVERSE|GROUP_CALL_REALTIME|GROUP_CALL_MULIILEVEL,"unhandled_input","_unhandled_input",ev);
 	if (!is_input_handled() && Object::cast_to<InputEventKey>(*ev) != nullptr) {
-		get_tree()->_call_input_pause(unhandled_key_input_group, "_unhandled_key_input", ev, this);
+		// Check all parents of focused GUI node first.
+		Variant arg = ev;
+		const Variant *v[1] = { &arg };
+		Node *node = _gui_get_focus_owner();
+		while (node && (node = node->get_parent())) {
```

Comment:
Won't this miss calling `_unhandled_key_input` on the actual `_gui_get_focus_owner()` node?

Reply:
Yep it will... I noticed this Shem writing it but forgot to fix. I guess I shouldn't rush to get PRs I'm before bed!